thoth/common/workflows.py

```diff
@@ -861,6 +861,27 @@ def submit_build_analysis(
 
         return workflow_id
 
+    def submit_sync_job(
+        self,
+        template_parameters: Optional[Dict[str, str]] = None,
+        workflow_parameters: Optional[Dict[str, Any]] = None,
+    ) -> Optional[str]:
+        """Submit graph-sync workflow."""
+        if not self.openshift.backend_namespace:
+            raise ConfigurationError("Backend namespace was not provided.")
+
+        template_parameters = template_parameters or {}
+        workflow_parameters = workflow_parameters or {}
+
+        workflow_id: Optional[str] = self.submit_workflow_from_template(
+            self.openshift.backend_namespace,
+            label_selector="template=sync",
```

Comment:
```suggestion
            label_selector="template=sync-job",
```

I feel like we should keep this more explicit, as `sync` might come up somewhere else as well.
I understand other components are not adding job suffix, however it might be good to add it for just this one.

Reply:
I can make this change, but I put it as sync because I went to go find the sync job in Openshift and it had the label `sync` rather than `sync-job`. Should I change the label in openshift to also reflect this? See [Smaug](https://console-openshift-console.apps.smaug.na.operate-first.cloud/k8s/ns/thoth-graph-prod/jobs) for more info on this.

Reply:
yes that would be great.